forte/utils/tagging_scheme.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Utility functions related to tagging scheme.
+"""
+
+import logging
+from typing import Optional, List, Union, Tuple
+
+
+def bio_merge(
+    tags: List[str],
+    types: List[Union[str, None]],
+    index: Optional[List[Tuple[Union[int, None], Union[int, None]]]] = None,
+) -> Tuple[
+    List[Union[str, None]], List[Tuple[Union[int, None], Union[int, None]]]
```

Comment:
The second return element: `result_indices`, should not contain `None`. It should be `List[Tuple[int, int]]`